client/app/components/dialog-components-static/dialog-components-static.component.js

```diff
@@ -0,0 +1,179 @@
+(function() {
+  'use strict';
+
+  // works as placeholders for drag&drop dialog components
+
+  angular.module('app.components')
+    .component('dialogComponentsStatic', {
+      controller: function() {
+        this.components = {
+          dialogFieldTextBox: {
+            icon: "fa-font",
+            label: __("Text Box"),
+            placeholders: {
```

Comment:
Hmm, looking at these, I'm starting to wonder whether these objects shouldn't have their own class (and a common superclasss) .. on the one hand it's nice to see it all together here, but on the other, there's lots of repetition and should the list ever grow, it may become ..unsavory :). Any idea if it will grow though?